src/main/java/org/apache/maven/archetypes/App.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.maven.archetypes;
+
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class App 
+{
```

Comment:
Bad formatting. Open block sign `{` on new line is c# style. In Java code convention should be on same raw as declaration.
Please look in your IDE settings Keymap and find `Reformat Code` hot key. Apply it every time before commit.

Reply:
done